src/homeblks/home_blks.cpp

```diff
@@ -634,6 +634,12 @@ bool HomeBlks::verify_index_bm() {
     return true;
 }
 
+uint64_t HomeBlks::get_used_volume(const boost::uuids::uuid& uuid) {
+    /* Update per volume status */
+   VolumePtr vol = lookup_volume(uuid);
```

Comment:
you can just assert here I think. return 0 can either mean there is 0 used size.

Reply:
if there is no volume found, `HomeBlksHttpServer::get_utilization()` return an error message  `vol not found!`. This is for safety (an edge case)that if between calling HomeBlksHttpServer::get_utilization Line 165 and 172, the volume does not exist in lookup map.

Reply:
if so, can we pass the volptr to homeblks directly since you've already looked it up, and no need to worry about lookup again and check the result.

Reply:
done